docs/quick_start/platforms.rst

```diff
@@ -31,6 +31,6 @@ Fledge can be built or installed in one of the following Linux distributions:
       - N/A |br| N/A
     * - Raspberry Pi OS (buster)
       - armv7l
-      - http://archives.fledge-iot.org/latest/bullseye/aarch64
+      - http://archives.fledge-iot.org/latest/buster/armv7l
       - N/A |br| N/A
```

Comment:
is there a way to show hr or line separators; that appears as separate cell next to merged cell?